package.json

```diff
@@ -66,6 +66,8 @@
         "bootstrap-select": "^1.13.16",
         "bootstrap-vue": "^2.15.0",
         "es6-promise": "^4.2.8",
+        "laravel-mix-merge-manifest": "^0.1.2",
```

Comment:
@dennis95stumm Do you know whether or not we need this?

Reply:
We don't need this any more!